CONTRIBUTING.md

```diff
@@ -92,16 +94,34 @@ $ANDROID_SDK_ROOT/cmdline-tools/latest/bin/sdkmanager --licenses
 Download Android system images for the minimum and targeted API levels. Note that you may get a warning about a `.cfg` file not being present. You're safe to ignore this.
 
 ```
-$ANDROID_SDK_ROOT/cmdline-tools/latest/bin/sdkmanager "system-images;android-28;google_apis;x86"
+# x86
+$ANDROID_SDK_ROOT/cmdline-tools/latest/bin/sdkmanager "system-images;android-29;google_apis;x86"
 $ANDROID_SDK_ROOT/cmdline-tools/latest/bin/sdkmanager "system-images;android-24;google_apis;x86"
+
+# ARM
+$ANDROID_SDK_ROOT/cmdline-tools/latest/bin/sdkmanager "system-images;android-29;google_apis;arm64-v8a"
+$ANDROID_SDK_ROOT/cmdline-tools/latest/bin/sdkmanager "system-images;android-24;google_apis;arm64-v8a"
```

Comment:
Wondering if we need to go as low as Andorid 24 and we should set our minimum to the Skyscanner minimum @marianeum can you confirm the Skyscanner base version or if we should do this?

Whats Backpack Androids settings?

Reply:
Yep 24 is the current minimum, so that's correct.